### PR TITLE
Feature/parent footer

### PR DESCRIPTION
**Pull request by milklot**
Create parent footer.

File path: src/components/common/ParentFooter.js
Created parent footer, added redirection to the existing pages. Some of them are still missing ( "Privacy" and "Terms")

File path: src/styles/less/ParentFooter.less
Added styling according to figma. 

File path: 
src/components/pages/AddChild/RenderAddChild.js
src/components/pages/EditPlayers/RenderEditPlayers.js
src/components/pages/Help/Help.js
src/components/pages/NewParentDashboard/RenderNewParentDashboard.js 
src/components/pages/ParentContact/ParentContactContainer.js
src/components/pages/ParentFaq/ParentFaqContainer.js

Added footer component.

***
There's some bug (fixed) - if component page need a scroll to get to the bottom of the page - footer will shows in the middle of the component. I added "scroll check" param, which changing className of the footer component if need/no need to scroll and changing .less styles of it.
So, basically, with adding ParentFooter component if no need to scroll the get the bottom of the page - add "layoutContainerCheck={'no-scroll'}"
(Example: `<ParentFooter layoutContainerCheck={'no-scroll'}` />)

Trello card: https://trello.com/c/qCjh7qL3/688-create-parent-footer

Let me know if you have any suggestions / recommendations for it, thanks

### CODE REVIEW

**Comment by milklot** [comment]
Screen Shot 2021-11-02 at 20 20 14

See attached screenshot of the bug, it's fixed, just for reference

**Comment by milklot** [comment]
Screen Shot 2021-11-02 at 20 14 41

Screen Shot 2021-11-02 at 21 08 25


Attached screenshot of the bug. Fixed (see second screenshot), just for reference

File: src/styles/less/ParentFooter.less [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/* Footer Styles */
+
+.no-scroll {
+	position: absolute;
+	bottom: 0;
+	width: 100%;
+}
+
+footer {
+	// position: absolute;
```
Comment (rhiannonstanford) [inline src/styles/less/ParentFooter.less:10]: Great job catching the bug here.  Absolute renders differently depending on how each component is styled.

**Review by rhiannonstanford — DISMISSED** [review]
Nice work filling in the gap by creating the parent footer.

**Review by artofmayhem — DISMISSED** [review]
Nice changes and additions setting this footer up. One step closer... no pun Intended

**Comment by bipin-shrestha** [comment]
Thanks for working in it and making it better.

**Review by rhiannonstanford — DISMISSED** [review]
Thank you for cleaning up those merge conflicts.

File: src/components/pages/ParentFaq/ParentFaqContainer.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { CaretRightOutlined } from '@ant-design/icons';
 import img1 from './img1.png';
 import ParentNavTopBar from '../../common/ParentNavTopBar';
 import ParentDashboardBack from '../../common/ParentDashboardBack';
+import ParentFooter from '../../common/ParentFooter';
```
Comment (Julianisrel) [inline src/components/pages/ParentFaq/ParentFaqContainer.js:7]: great job looks good.

**Review by Julianisrel — DISMISSED** [review]

**Review by rhiannonstanford — APPROVED** [review]
Looks good.  Conflicts fixed.

**Review by ghost — APPROVED** [review]
Great work creating the parent footer!
